src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -305,6 +306,20 @@ def _gen_paths(cls, all_endpoints):
                                 'application/json': {
                                     'schema': cls._gen_schema_for_content(body_params)}}}
 
+                    if endpoint.query_params:
+                        query_params = cls._add_param_info(endpoint.query_params, p_info)
+                        methods[method.lower()]['requestBody'] = {
+                            'content': {
+                                'application/json': {
+                                    'schema': cls._gen_schema_for_content(query_params)}}}
+
+                    if endpoint.path_params:
+                        path_params = cls._add_param_info(endpoint.path_params, p_info)
+                        methods[method.lower()]['requestBody'] = {
+                            'content': {
+                                'application/json': {
+                                    'schema': cls._gen_schema_for_content(path_params)}}}
+
```

Comment:
Why is this only for `POST` and `PUT`?

Reply:
As per the previous implementation, the **request body schema** was there only for POST and PUT operations, but it was there only for **body parameters** of the endpoint, I added it for the **query params** and the **path params** as well. Moreover I was following bluejeans API documentation as reference (https://bluejeans.github.io/api-rest-meetings/site/index.html), which also has **request body schema** for the PUT and POST operations only, not for GET or DELETE operations. If still required we can add for the GET and DELETE operations as well.

Reply:
Nope, what I mean is that I don't it's correct to include in the request body the params that belong to the URL. In the following example we see that the `fs_id` is sent duplicated (in the URL and in the request body):

![image](https://user-images.githubusercontent.com/37327689/89328530-d164d300-d68d-11ea-9eb2-31dfc1b6d02d.png)

Reply:
Corrected this.